### ISSUE TITLE
Wrong value for ENVELOPE_NAME and BASE_TYPE in PageViewTelemetry.java

### ISSUE THREAD

**Issue by astecenko** [Microsoft/ApplicationInsights-Java#497]
In [PageViewTelemetry.java](https://github.com/Microsoft/ApplicationInsights-Java/blob/master/core/src/main/java/com/microsoft/applicationinsights/telemetry/PageViewTelemetry.java) we see:

> private static final String ENVELOPE_NAME = "PerformanceCounter";
> private static final String BASE_TYPE = "PerformanceCounterData";

but based on .Net SDK need:

> private static final String ENVELOPE_NAME = "PageView";
> private static final String BASE_TYPE = "PageViewData";

**Comment by dhaval24** [comment]
@astecenko yes we have seen this discrepancy. Is the PageView Telmetry working for you using AI-Java SDK ? Is this causing any breaking issues ?

**Comment by astecenko** [comment]
I do not use the AI-Java SDK in my projects, and I did not test the PageView Telemetry work. I develop projects in Embarcaderro Delphi, and took the AI-Java SDK as the basis for developing the AI-Delphi SDK that is needed for my projects. During development, I saw this discrepancy in the meaning of constants.

AI-Java SDK is a fully functional SDK? Or is it in development, and I made the mistake of selecting it as the basis for the AI-Delphi SDK?

**Comment by dhaval24** [comment]
@astecenko I am not sure what AI-Delphi SDK is about, atleast never heard of it. However to answer your question,  AI-SDK is a fully functional SDK with basic monitoring capacity. Sure it has some gaps with .NET SDK however, with all the latest changes currently in the master those gaps are on smaller end. You can get most of the monitoring functionality with the SDK.  of-course the development is continuous process, we want to keep iterating, fixing issues and introduce new features, but I think there is no mistake on your end. SDK is very well extensible. Also if you find any issues with the SDK please do feel free to raise PR if you wish to.

**Comment by tobiasdiez** [comment]
@dhaval24 For us, tracking page views is currently not possible with 1.10 and 2.0 beta (but 1.9 works). Originally, I thought that is was part of https://github.com/Microsoft/ApplicationInsights-Java/issues/448#issuecomment-337173300 but probably the wrong constants here are the reason as the problem still persists with the 2.0 beta. Maybe not the sole reason since the change https://github.com/Microsoft/ApplicationInsights-Java/commit/f36a0b5820fcf19e7b297da17648038774ed645b#diff-392cf77c5af2e23ef895e54919ea9e45 of the constants occurred before the release of 1.09.

**Comment by dhaval24** [comment]
@tobiasdiez thanks for pointing this our. Probably the wrong constants here may be the possible reason. I will try by fixing the constants and see if it resolves the issue. The comment on #448 was generally for severity level issue.

**Comment by dhaval24** [comment]
@tobiasdiez @astecenko  just submitted a PR for this fix. #510 . Also performed an E2E test and saw the page view telemetry data flowing in. As far as I think the jars for 1.0.9 were build way before the incorrect constants were pushed in and hence @tobiasdiez it did not probably had the changes you mentioned all though they were merged into the master.

**Comment by dhaval24** [comment]
@tobiasdiez this changes are now in Master. If needed you can pull the master and build it locally to see if things are now working for you.

**Comment by tobiasdiez** [comment]
Thanks for the quick fix. I'm not sure if I manage to find the time to build the new master myself but we'll definitely try out the next released version (do you have a snapshot channel on maven?).

**Comment by dhaval24** [comment]
@tobiasdiez unfortunately we do not have a snapshot channel on Maven. Apologies, but I do not have more idea about this. Could you tell me how we can set one up? From brief knowledge I think its more about enabling people to download latest builds which might become stable.

**Comment by dhaval24** [comment]
@grlima @littleaj the fix for this was relatively very small (i.e only constant change) but affect is it unblocks the whole feature of our product. Should we cherry pick this for our stable release? I do not think it would cause any performance regression.
